tests/common/hparams.py

```diff
@@ -0,0 +1,30 @@
+from typing import Any, Callable, Dict, Optional, Sequence, Type
+
+import yahp as hp
+import yaml
+
+
+def assert_registry_contains_entry(subclass: Type, registry: Dict[str, Callable], ignore_list: Sequence[Callable] = ()):
+    """Assert that the ``registry`` contains """
+    registry_entries = set(registry.values())
+    registry_entries.union(ignore_list)
+    assert subclass in registry_entries, f"Class {type(subclass).__name__} is missing from the registry."
+
+
+def assert_is_constructable_from_yaml(
```

Comment:
`assert_yaml_builds`?

Reply:
Makes sense; decided to go with `assert_yaml_loads` instead (`builds` seems a _bit_ too much for what we're doing with the yaml, since we're not necessarily calling `.initialize_object()` here.)